### ISSUE TITLE
List of potential examples to include in this repo

### ISSUE THREAD

**Issue by MichaelClifford** [aicoe-aiops/data-science-workflow-examples#4]
As I data scientists, I would like to have a library of general purpose and reusable notebooks, on a variety of Data Science topics so that as a team we are able to share, reference and re-use fundamental data science work to make us more efficient.    

List of topics:

- [ ] Dimensional Reduction
- [ ] Explainability  
- [ ] Time Series Anomaly Detection (univariate/ multidimensional) 
- [ ] Feature Engineering/ Feature Selection

**Comment by Shreyanand** [comment]
+1 on the idea. Do we want to add details on what aspects these notebooks should focus on?
For e.g., pros and cons of each dim. red. method...

**Comment by MichaelClifford** [comment]
@Shreyanand good point. We can edit the issue to include some more details/ guidelines on what the notebooks should focus on.
